Source/DataSources/GeoJsonDataSource.js

```diff
@@ -690,6 +690,12 @@ define([
         name : {
             get : function() {
                 return this._name;
+            },
+            set : function(value) {
```

Comment:
OK, looking at the code we don't seem to use this internally other than setting the DataSource clock in viewer, so this is fine as is.